docs/advanced-topics/ideal_integration.md

```diff
@@ -41,7 +41,8 @@ arise as source code changes over time.
 
 Make sure to fuzz the target locally for a small period of time to ensure that 
 it does not crash, hang, or run out of memory instantly.
-See details at http://libfuzzer.info and http://tutorial.libfuzzer.info.
+You can read more about it [here]
+(https://github.com/google/fuzzer-test-suite/blob/master/tutorial/libFuzzerTutorial.md#problems)
```

Comment:
Would https://github.com/google/fuzzing/blob/master/docs/good-fuzz-target.md be even a better destination?

Reply:
Done!